lib/services/common-api-presenter.js

```diff
@@ -237,6 +237,16 @@ function CommonApiPresenterFactory(
                                                         config.apiServerPort ||
                                                         80;
         var serverUri = 'http://' + apiServerAddress + ':' + apiServerPort;
+        var fileServerUri;
+        if (_.has(config, 'fileServerAddress') &&
+            _.has(config, 'fileServerPort') &&
+            _.has(config, 'fileServerPath')) {
```

Comment:
I would suggest not making fileServerPath a mandatory field. If user meant to use the root directory of the file server, they would got a chance of not specifying the fileServerPath. 

This is a non blocking anyway.